pyproject.toml

```
@@ -44,6 +44,7 @@ dependencies = [
   "ruff",
   "semver~=3.0.2",
   "cookiecutter",
+  "click<=8.1.8"
```

Assuming comments here are valid in this file:

```suggestion
  # Temporarily pin until `typer` is compatible:
  # https://github.com/Comfy-Org/comfy-cli/issues/264#issuecomment-2871116559
  "click<=8.1.8"
```

Better for maintainers to be aware that this isn't a direct dep required by `comfy-cli`, and that it can be removed at a later date.

Ref: https://github.com/Comfy-Org/comfy-cli/issues/264#issuecomment-2878118243